packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -33,9 +33,16 @@ export default class QuanticResultLink extends LightningElement {
    *   - `_top`: the topmost browsing context (the "highest" context that’s an ancestor of the current one). If there are no ancestors, this behaves as `_self`.
    * @api
    * @type {string}
-   * @defaultValue `'_self'`
+   * @defaultValue `'_blank'`
```

Comment:
Uhm idk if we should change the default value in the resultLink component. Why not just set the `target` option to "_blank" from the quickview component that uses it?